compiler/daml-lf-proto/src/DA/Daml/LF/Proto3/DecodeV1.hs

```diff
@@ -164,13 +164,11 @@ decodeChoice LF1.TemplateChoice{..} =
     <*> mayDecode "templateChoiceRetType" templateChoiceRetType decodeType
     <*> mayDecode "templateChoiceUpdate" templateChoiceUpdate decodeExpr
 
--- FixMe: https://github.com/digital-asset/daml/issues/2289
---   drop the `error "Numeric not implemented"` in the following function
 decodeBuiltinFunction :: MonadDecode m => LF1.BuiltinFunction -> m BuiltinExpr
 decodeBuiltinFunction = pure . \case
   LF1.BuiltinFunctionEQUAL_INT64 -> BEEqual BTInt64
   LF1.BuiltinFunctionEQUAL_DECIMAL -> BEEqual BTDecimal
-  LF1.BuiltinFunctionEQUAL_NUMERIC -> error "Numeric not implemented"
+  LF1.BuiltinFunctionEQUAL_NUMERIC -> BEEqualNumeric
```

Comment:
Do we have tests for the decoder anywhere? Maybe some roundtrip tests?

Reply:
I couldn't find any. I added the issue #2682 to track this.